lms/static/js/groups/views/cohorts.js

```diff
@@ -304,7 +304,7 @@
                                 });
                             }
                         }).render();
-                        this.$('#file-upload-form-file').focus();
+                        this.$('#file-upload-form-file');
```

Comment:
By focusing on the form field, the a11y checks weren't able to find the focusable target.